src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -28,15 +28,12 @@ uct_dc_mlx5_iface_bcopy_post(uct_dc_mlx5_iface_t *iface, uct_dc_mlx5_ep_t *ep,
     UCT_DC_MLX5_TXQP_DECL(txqp, txwq);
 
     UCT_DC_MLX5_IFACE_TXQP_GET(iface, ep, txqp, txwq);
-    desc->super.sn = txwq->sw_pi;
-    uct_rc_mlx5_txqp_dptr_post(&iface->super, UCT_IB_QPT_DCI, txqp, txwq,
-                               opcode, buffer, length, &desc->lkey,
-                               rdma_raddr, rdma_rkey, 0, 0, 0, 0,
-                               &ep->av, uct_dc_mlx5_ep_get_grh(ep),
-                               uct_ib_mlx5_wqe_av_size(&ep->av),
-                               MLX5_WQE_CTRL_CQ_UPDATE | send_flags, imm_val_be, INT_MAX,
-                               log_sge);
-    uct_rc_txqp_add_send_op(txqp, &desc->super);
+    uct_mlx5_txqp_bcopy_post(&iface->super, UCT_IB_QPT_DCI, txqp, txwq, opcode,
```

Comment:
do we really need `uct_dc_mlx5_iface_bcopy_post` wrapper now? Seems can just use `uct_mlx5_txqp_bcopy_post` and declare `txqp, txwq` in place